src/sac_stac/service_layer/services.py

```diff
@@ -227,6 +227,10 @@ def add_stac_item(repo: S3Repository, acquisition_key: str, update_collection_on
 
 
 def create_geom(geometry, crs):
-    poly = GeoSeries([geometry.exterior], crs=crs).to_crs(GENERIC_EPSG).to_json()
+
+    if isinstance(geometry, Polygon):
```

Comment:
This could just be a known issue with Shapely==1.7.1. Can we try retry with Shapely==1.8.4.

Or is this fix fine?

Reply:
Id rather not bump versions right now, that might break other things.